src/Package/Test/DataInspect/REnvironmentProviderTest.cs

```diff
@@ -20,63 +20,55 @@ namespace Microsoft.VisualStudio.R.Package.Test.DataInspect {
     [ExcludeFromCodeCoverage]
     [Collection(CollectionNames.NonParallel)]   // required for tests using R Host 
     public class REnvironmentProviderTest {
-        private ManualResetEvent _environmentChanged = new ManualResetEvent(false);
-        private REnvironmentCollection _environments;
-
         [Test]
         [Category.Variable.Explorer]
         public async Task GetREnvironmentTest() {
-            await RunEnvironmentTestAsync("1", () => {
-                _environments.Count.Should().BeGreaterOrEqualTo(2);
-                _environments[0].Name.Should().BeEquivalentTo(".GlobalEnv");
-                _environments[_environments.Count - 1].Name.Should().BeEquivalentTo("package:base");
+            await RunEnvironmentTestAsync("1", (environments) => {
+                environments.Count.Should().BeGreaterOrEqualTo(2);
+                environments[0].Name.Should().BeEquivalentTo(".GlobalEnv");
```

Comment:
(and the same everywhere below, except where you need case-insensitive, leading/trailing whitespace ignoring comparisons)